js/weatherWidget.js

```diff
@@ -0,0 +1,38 @@
+// Fetches weather data and updates - External API documentation: https://rapidapi.com/community/api/open-weather13
+
+
+async function fetchWeather() {
+    const url = 'https://open-weather13.p.rapidapi.com/city/sydney';
+    const options = {
+        method: 'GET',
+        headers: {
+            'X-RapidAPI-Key': '3175b02e8cmsh130ca381e03b97fp13191ajsna842d953489 a', // remember to remove the space before the last 'a', the space is for saving my monthly request limit
```

Comment:
You should document this in the readme incase anyone is looking through and thinks you made an error with the code

Reply:
^ Or leave instructions for them to get their own key.